packages/components/inputs/search-select-input/src/search-select-input.js

```diff
@@ -0,0 +1,151 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import { ThemeContext } from '@emotion/core';
+import { SafeHTMLElement } from '@commercetools-uikit/utils';
+import { SearchIconDropdownIndicator } from '@commercetools-uikit/select-utils';
+import AsyncSelectInput from '@commercetools-uikit/async-select-input';
+import messages from '../../../../../src/components/internals/messages/select';
+import createSelectStyles from '../../../../../src/components/internals/create-select-styles';
+
+const useTheme = () => useContext(ThemeContext);
+
+const SearchSelectInput = (props) => {
+  const intl = useIntl();
+  const theme = useTheme();
+  const noOptionsMessageWithOutInputValue = props.initialSuggestionMessage
+    ? props.initialSuggestionMessage()
+    : intl.formatMessage(messages.noOptionsMessageWithoutInputValue);
+  const noOptionsMessageWithInputValue = props.noOptionsMessage
+    ? props.noOptionsMessage()
+    : intl.formatMessage(messages.noOptionsMessageWithInputValue);
+
+  return (
+    <AsyncSelectInput
+      horizontalConstraint={props.horizontalConstraint}
+      // react-select uses "id" (for the container) and "inputId" (for the input),
+      // but we use "id" (for the input) and "containerId" (for the container)
+      // instead.
+      // This makes it easier to less confusing to use with <label />s.
+      id={props.containerId}
+      inputId={props.id}
+      aria-label={props['aria-label']}
```

Comment:
We could use `{...filterAriaAttributes(props)}` for these.

I know the other SelectInputs don't do it but they should, so let's already simplify this one 😄